assignment4/assignment4.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+# Task 1.
+data = {
+    'Name': ['Alice', 'Bob', 'Charlie'],
+    'Age': [25, 30, 35],
+    'City': ['New York', 'Los Angeles', 'Chicago']
+}
+
+task1_data_frame = pd.DataFrame(data)
+print(task1_data_frame)
+
+task1_with_salary = task1_data_frame.copy()
+task1_with_salary["Salary"] = [70000, 80000, 90000]
+print(task1_with_salary)
+
+task1_older = task1_with_salary.copy()
+task1_older['Age'] += 1
+print(task1_older)
+
+task1_older.to_csv('employees.csv', index=False)
+
+
+# Task 2.
+task2_employees = pd.read_csv('employees.csv')
+print(task2_employees)
+
+json_employees = pd.read_json('additional_employees.json')
+print(json_employees)
+
+more_employees = pd.concat([task2_employees, json_employees], ignore_index=True)
+print(more_employees)
+
+
+# Task 3.
+first_three = more_employees.head(3)
+print(first_three)
+
+last_two = more_employees.tail(2)
+print(last_two)
+
+employee_shape = more_employees.shape
+print(employee_shape)
+
+print(more_employees.info())
```

Comment:
This is mostly fine, but note that if you print `df.info()` you will get `None` in addition to the info. This is a quirk of how info() method works: `info()`'s sole purpose is to print the info about the dataframe, so you don't have to print(info()). In fact, it returns `None`, so if you do print it, it will print out `None`.  

This is a minor thing.